analysis/topology/topological_structures.lean

```diff
@@ -254,6 +254,10 @@ instance topological_ring.to_topological_semiring
 instance topological_ring.to_topological_add_group
   [topological_space α] [ring α] [t : topological_ring α] : topological_add_group α := {..t}
 
+/-- A topological commutative ring is a commutative ring where the ring operations are continuous. -/
+class topological_comm_ring (α : Type u) [topological_space α] [comm_ring α]
```

Comment:
This is not necessary. In fact this is exactly what is **not** necessary when using the parameter style.

You just write
```lean
lemma f [topological_space α] [comm_ring α] [topological_ring α] ...
```
end it should workw

Reply:
Removed, thanks.